sdk/tables/Azure.Data.Tables/tests/TableClientTests.cs

```diff
@@ -426,12 +426,7 @@ public async Task ValidateUri()
         [Test]
         public void CreateIfNotExistsThrowsWhenTableBeingDeleted()
         {
-            _transport = new MockTransport(
-                request => throw new RequestFailedException(
-                    (int)HttpStatusCode.Conflict,
-                    null,
-                    TableErrorCode.TableBeingDeleted.ToString(),
-                    null));
+            _transport = TableAlreadyExistsTransport(TableErrorCode.TableBeingDeleted);
```

Comment:
Where do you validate that errors are not logged/error traces are not sent?

Reply:
My assumption here is that this is guaranteed by the `RequestContext.AddClassifier` behavior. I think I'd have to test the Core event source to validate either way.

Reply:
Validating the event source may be more trouble than it's worth, given that you'd have to set this up in a non-parallelized test, use the helper from the test framework, and hope that no unrelated error was logged, since there doesn't seem to be anything specific to Tables that you could key on.

Personally, I'm good with trusting the classifier and assuming that it has robust test coverage.